### PR TITLE
Search shows NO RESULTS AVAILABLE if no matching zim files exists

### PR DESCRIPTION
**Pull request by ritik2017**
Fixes: Issue #1070 

Changes: 
1, TextView showing No Results Available made when no matching zim files is available
2. The Visibility of TextView is set as the search results in currentAdapter

Screenshots/GIF for the change: [If possible, please add relevant screenshots/GIF]
ezgif com-video-to-gif (4)

### CODE REVIEW

**Comment by ritik2017** [comment]
@abdulwd @mhutti1 Can you please review the request ? 
Do you think we should change the error message or No Results Available is fine

**Comment by ritik2017** [comment]
@abdulwd The code compiles well in android studio and runs fine in the device 
But travis-ci shows the error
/home/travis/build/kiwix/kiwix-android/app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java:94: error: cannot find symbol
import org.kiwix.kiwixmobile.data.local.entity.Bookmark;

Please Help!!
Thanks

File: app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java [hunk 1]
```diff
@@ -70,11 +70,13 @@
   private SearchView searchView;
   private String searchText;
 
+
```
Comment (iadeelzafar) [inline app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java]: This extra line can be removed.

File: app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java [hunk 2]
```diff
@@ -70,11 +70,13 @@
...
   private TextView errorMessage_textview;
 
   @Inject
   SearchPresenter searchPresenter;
 
+
```
Comment (iadeelzafar) [inline app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java]: Here too.

File: app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java [hunk 3]
```diff
@@ -89,6 +96,9 @@ protected void onCreate(Bundle savedInstanceState) {
     searchPresenter.getRecentSearches();
     activateDefaultAdapter();
 
+    errorMessage_textview = (TextView) findViewById(R.id.error_search);
```
Comment (abdulwd) [inline app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java]: Use butter knife here.

File: app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java [hunk 4]
```diff
@@ -89,6 +96,9 @@ protected void onCreate(Bundle savedInstanceState) {
...
+    errorMessage_textview.setVisibility(View.GONE);
```
Comment (abdulwd) [inline app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java]: Do this in XML.

**Comment by abdulwd** [comment]
Closing due to inactivity.

**Pull request closed without merging**